README.md

```diff
@@ -14,88 +14,110 @@ Python 3.7 and above are supported.
 ## How to use
 
 ### Cloud
-```python
-from pyrisco import RiscoCloud
-r = RiscoCloud("<username>", "<password>", "<pincode>")
-
-# you can also pass your own session to login. It will not be closed
-await r.login()
-alarm = await r.get_state()
-# partitions and zones are zero-based in Cloud
-print(alarm.partitions[0].armed)
-
-events = await r.get_events("2020-06-17T00:00:00Z", 10)
-print(events[0].name)
-
-print(alarm.zones[0].name)
-print(alarm.zones[0].triggered)
-print(alarm.zones[0].bypassed)
 
-# arm partition 0
-await r.partitions[0].arm()
+#### RiscoCloud fallback mode
 
-# and disarm it
-await r.partitions[0].disarm()
+Pyrisco will instruct RiscoCloud to request your control panel, if there is an issue RiscoCloud will return a 72 error code, if this happens:
+* pyrisco will try a second time in fallback mode, this will request the last known state from RiscoCloud.
+* a flag `assumed_control_panel_state` will be set to True on the Alarm object to indicate that the state is assumed.  
```

Comment:
```suggestion
* A flag named `assumed_control_panel_state` will be set to True on the Alarm object to indicate that the state is assumed, rather than obtained from the panel. **Assumed states could be stale.**
```